recipes/jhsingle-native-proxy/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "jhsingle-native-proxy" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name | replace('_', '-') }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 69c57349856db0887b23c476491c54d2dfc947de310cd331be3aef5dc8e6bb0f 
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  entry_points:
+    - jhsingle-native-proxy = jhsingle_native_proxy.main:run
+
+requirements:
+  host:
+    - pip
+    - python
```

Just to be coherent with ``run``, let's add the python constrains here as well :)
```suggestion
    - python >=3.6.0,<4.0.0
```